.github/workflows/XRefCheck.yml

```diff
@@ -0,0 +1,15 @@
+name: XRefCheck
```

Comment:
This action checks links to relative files (e.g. `/images/tralala.png`) and the internal links withing an md-document (e.g. `#5-references`). External links (e.g. `http://tralala.com`) are theoretically checked as well, but not as good as by another checker (some wrong external links are not shown)